src/classes/LD_LeadConvertOverride_CTRL.cls

```diff
@@ -424,17 +424,17 @@ public with sharing class LD_LeadConvertOverride_CTRL {
                     ID rtIdAcccount = ACCT_IndividualAccounts_TDTM.rtIdForNewAccount();
                     if (rtIdAcccount != null) 
                         myAccount.put('RecordTypeID', rtIdAcccount);
-                    myAccount.npe01__SYSTEMISINDIVIDUAL__c = true;
-                    myAccount.npe01__One2OneContact__c = con.Id;
-                    update myAccount;
+                        myAccount.npe01__SYSTEMISINDIVIDUAL__c = true;
```

Comment:
the if on line 425 was only for 426, not 427-429.  so rather than indenting 427-429, let's add the recommended bracing on our If.  thanks!

Reply:
Braces added and indentation fixed. Thanks!